packages/aws-cdk-lib/aws-cloudfront/lib/function.ts

```diff
@@ -212,3 +218,19 @@ export interface FunctionAssociation {
   /** The type of event which should invoke the function. */
   readonly eventType: FunctionEventType;
 }
+
+/**
+ * The function's runtime environment version.
+ */
+export enum FunctionRuntime {
```

Comment:
I think this would be better as an enum-like class, similar to how our other Rumtime types look.